tutorial/tutorial08.yml

```diff
@@ -28,6 +36,14 @@ cables:
     manufacturer: [WiresCo,WiresCo,WiresCo,WiresCo] # set a manufacter per wire
```

Comment:
This is an issue that might have to be considered more generally, perhaps in the context of #56 and #127, where you might want other properties assigned on a per-wire basis.
Just something to keep in mind. I still think @kvid's suggestion could come in handy here.

Reply:
My comment above is not about changing any feature. This is a feature that has been implemented several months ago and is present in v0.2. Wire attributes `pn`, `manufacturer`, and `mpn` can all be specified different for each wire in a list, or as a single value common to all wires. Please correct me if I'm wrong. See inserting [common values](https://github.com/formatc1702/WireViz/blob/v0.2/src/wireviz/Harness.py#L163-L166) and [list entries](https://github.com/formatc1702/WireViz/blob/v0.2/src/wireviz/Harness.py#L202-L206) in the diagram, and [inserting either](https://github.com/formatc1702/WireViz/blob/v0.2/src/wireviz/Harness.py#L379-L381) in the BOM.

My suggested comment in the tutorial is just to make the user aware of this feature, but please suggest a different/better way to demonstrate this feature. I'm sorry that the [doc](https://github.com/formatc1702/WireViz/blob/v0.2/docs/syntax.md#cable-attributes) slipped through my review without mentioning this feature.

If adding such a comment is a big issue that needs a discussion, then it's out of scope here. Adding a similar feature for other attributes is definitely out of scope here. 😃

Reply:
Looking back at the code, there is actualy different functionality in the graph depening on if it is defined as a list or as a sting. If it is a string it will be listed in the top info of a bundle (like it is for a cable), or if it is a list it is listed per wire. The output can be altered so bundles are always rendered per wire even if it is a string, this is probably worth doing as it will avoid bundles looking incosistent.

Reply:
@Tyler-Ward has been so kind and updated `syntax.md` to reflect the changes of this PR already, so I would request to add a note regarding this functionality (parameters accepting either `str` or `List[str]`) as well :)

Reply:
Sorry, it seems we were typing a comment at the same time.

> The output can be altered so bundles are always rendered per wire even if it is a string, this is probably worth doing as it will avoid bundles looking incosistent.

Let's move that discussion to a new issue/PR so as not to delay this one too much, since it's drifting out of scope a bit. Based on my recent thoughts posted in #56 we'll need to think about the general issue of how to deal with parameters that differ between wires of a bundle, or perhaps even within a regular cable (e.g. different gauges).
So please don't worry about this point right now.

Reply:
Sounds sensible, will open an issue for this now.